app/assets/javascripts/float-kit/addon/modifiers/apply-floating-ui.js

```diff
@@ -0,0 +1,109 @@
+import Modifier from "ember-modifier";
+import { registerDestructor } from "@ember/destroyable";
+import { bind } from "discourse-common/utils/decorators";
+import {
+  arrow,
+  autoPlacement,
+  autoUpdate,
+  computePosition,
+  inline,
+  offset,
+  shift,
+} from "@floating-ui/dom";
+import { headerOffset } from "discourse/lib/offset-calculator";
+import { iconHTML } from "discourse-common/lib/icon-library";
+import { cached } from "@glimmer/tracking";
+import domFromString from "discourse-common/lib/dom-from-string";
+import { FLOAT_UI_PLACEMENTS } from "float-kit/lib/constants";
+
+export default class FloatKitApplyFloatingUi extends Modifier {
+  constructor(owner, args) {
+    super(owner, args);
+    registerDestructor(this, (instance) => instance.teardown());
+  }
+
+  modify(element, [trigger, options]) {
+    this.element = element;
+    this.trigger = trigger;
+    this.options = options ?? {};
+
+    if (this.options.arrow) {
+      element.appendChild(this.arrowElement);
+    }
+
+    if (this.options.autoUpdate) {
+      this.cleanup = autoUpdate(trigger, element, this.update);
+    } else {
+      this.update();
+    }
+  }
+
+  @cached
+  get arrowElement() {
+    return domFromString(
+      iconHTML("tippy-rounded-arrow", { class: "arrow" })
```

Comment:
I see this is just what we have called the SVG...but since we are getting rid of tippy maybe we just rename it to `floatkit-arrow` or something?

Reply:
Yes I didnt do it for now because there are places calling it directly